rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -326,7 +326,7 @@ public static ErpFederation deserializeErpFederation(
             BridgeSerializationUtils::deserializeFederationMember
         );
 
-        return new ErpFederation(
+        return new LegacyErpFederation(
```

Comment:
Same here, I guess this method is in charge of deserializing a LegacyErpFederation right? In that case, I'd change the name, parameter and probably the return type also

Reply:
But please check first if that is the case

Reply:
yes, you are right. there are some names to change